<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>898</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="collection_layout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_26">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="collection_label">
       <property name="text">
        <string>Collection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="collectionName_comboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="collectionSelect_pushButton">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_27">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="objectAttribute_layout">
     <item>
      <layout class="QVBoxLayout" name="object_layout">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QFrame" name="object_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="objectAddRemove_layout">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="object_label">
              <property name="text">
               <string>Objects</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="objectAdd_toolButton">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="objectRemove_toolButton">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeView" name="object_treeView"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="attribute_layout">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QFrame" name="attribute_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="attributeAddRemove_layout">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="attribute_label">
              <property name="text">
               <string>Attributes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="attributeAdd_toolButton">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="attributeRemove_toolButton">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeView" name="attribute_treeView"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="solverOptions_layout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QFrame" name="solverOptions_frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="solverTop_layout">
          <item>
           <widget class="QLabel" name="solver_label">
            <property name="text">
             <string>Solvers</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="solverAdd_toolButton">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="solverRemove_toolButton">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="solver_layout">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QTableView" name="solver_tableView"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="solverSide_layout">
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="solverMoveUp_toolButton">
              <property name="text">
               <string>△</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="solverMoveDown_toolButton">
              <property name="text">
               <string>▽</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="emptySpace02_layout">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="emptySpace01_layout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
